src/browser/BrowserPasswordGeneratorWidget.cpp

```diff
@@ -0,0 +1,346 @@
+/*
```

Comment:
I think we should use PasswordGeneratorWidget. About the generate buttons and the password field we can use something like the standalone mode of the current widget. Like an http mode that will hide useless UI controls